app/experimenter/nimbus-ui/src/gql/experiments.ts

```diff
@@ -8,82 +8,17 @@ export const CREATE_EXPERIMENT_MUTATION = gql`
   mutation createExperiment($input: ExperimentInput!) {
     createExperiment(input: $input) {
       message
-      status
       nimbusExperiment {
-        name
         slug
-        hypothesis
-        application
       }
     }
   }
 `;
 
-export const UPDATE_EXPERIMENT_OVERVIEW_MUTATION = gql`
-  mutation updateExperimentOverview($input: ExperimentInput!) {
+export const UPDATE_EXPERIMENT_MUTATION = gql`
+  mutation updateExperiment($input: ExperimentInput!) {
     updateExperiment(input: $input) {
       message
-      status
-      nimbusExperiment {
-        name
-        hypothesis
-        publicDescription
-      }
-    }
-  }
-`;
-
-export const UPDATE_EXPERIMENT_STATUS_MUTATION = gql`
-  mutation updateExperimentStatus($input: ExperimentInput!) {
-    updateExperiment(input: $input) {
-      message
-      status
-      nimbusExperiment {
-        status
-      }
-    }
-  }
-`;
-
-export const UPDATE_EXPERIMENT_BRANCHES_MUTATION = gql`
-  mutation updateExperimentBranches($input: ExperimentInput!) {
-    updateExperiment(input: $input) {
-      message
-      status
-      nimbusExperiment {
-        id
-        featureConfig {
-          name
-        }
-        referenceBranch {
-          name
-          description
-          ratio
-        }
-        treatmentBranches {
-          name
-          description
-          ratio
-        }
-      }
-    }
-  }
-`;
-
-export const UPDATE_EXPERIMENT_PROBESETS_MUTATION = gql`
-  mutation updateExperimentProbeSets($input: ExperimentInput!) {
-    updateExperiment(input: $input) {
-      nimbusExperiment {
-        id
-        primaryProbeSets {
-          slug
-        }
-        secondaryProbeSets {
-          slug
-        }
```

Comment:
Hrmm, I see why you would combine these, but I would argue the general idea with GQL is you only request what you want back in a given query. I don't think having every field results in additional DB work, but the response will always be the largest possible response. In an app like this it's probably negligible, but probably a good practice to keep.

Reply:
Wait we aren't using any of the returned fields? Nvm ignore me

Reply:
Oh yeah we do a full refresh after each save anyway.

Reply:
Yep

Reply:
I'm trying to think of any reason why we should keep these separate, but coming up empty. Are they just retrofitted from when we had individual mutations?

Reply:
I think so.